File: readability.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>
#include <ctype.h>
#include <math.h>

int main(void)
{   //Get Input from User: String of Text
string text = get_string("Text: ");

int i;
int count_letters = 0;
int count_words = 1;
int count_sentences = 0;
    
    for (i=0; i < strlen(text); i++)
    {
        char c = text[i];
        if (isalpha (c))
        {
            count_letters++;
        }
        if (isspace (c))
        {
            count_words++;
        }
        if (c == '.' || c == '!' || c == '?')
        {
            count_sentences++;
        }
    }

    float L = (count_letters / count_words) * 100;
    float S = (count_sentences / count_words) * 100;
    float index = ((0.0588 * L) - (0.296 * S) - 15.8);
    int estimated_grade = round(index);

    if (estimated_grade >= 1 && estimated_grade < 16)
    {
        printf("Grade %i\n", estimated_grade);
    }
    else if (estimated_grade >= 16)
    {
        printf("Grade 16+\n");
    }
    else 
    {
        printf("Before Grade 1\n");
    }
}

    //Get Output: Grade "X"

    //If Output >=16, produce "Grade 16+"

    // If Output < 1, produce "Before Grade 1"

File: readability.py
from cs50 import get_string

text = get_string("Text: ")

## variables set
letters = 0
words = 1
sentences = 0

## Go through each character in text
for element in range(0, len(text)):
    if text[element].isalpha()==True:
        letters += 1
    if text[element].isspace()==True:
        words = words + 1
    if text[element] == '.' or text[element] == '?' or text[element] =='!':
        sentences += 1

L = (letters / words) * 100
S = (sentences / words) * 100

## Formula and Print Grade 
grade index = round(0.0588 * L - 0.296 * S - 15.8)
if grade index < 1:
    print ("Before Grade 1")
elif grade index < 16 and grade index >= 1:
    print (f"Grade {index}")
else:
    print ("Grade 16+")
    